libs/designer/src/lib/ui/panel/nodeDetailsPanel/nodeDetailsPanel.tsx

```diff
@@ -224,11 +224,11 @@ export const NodeDetailsPanel = (props: CommonPanelProps): JSX.Element => {
     return undefined;
   };
 
-  const getChildWorkflowIdFromInputs = (inputs: any): string | undefined => {
-    if (!isNullOrEmpty(inputs)) {
-      const workflow = getObjectPropertyValue(inputs, ['host', 'value', 'workflow']);
+  const getChildWorkflowIdFromInputs = (childWorkflowInputs: any): string | undefined => {
+    if (!isNullOrEmpty(childWorkflowInputs)) {
+      const workflow = getObjectPropertyValue(childWorkflowInputs, ['host.workflow.id']);
```

Comment:
just wondering is host.workflow.id a path? should it be done through: 
const workflow = getObjectPropertyValue(childWorkflowInputs, ['host', 'workflow', 'id']);

Reply:
The reason on why this is like this, is because when we introduced the InputsBinder we started to render and building the inputs/outputs schema based on the manifest/swagger. This is the manifest from the workflow type action
<img width="1433" alt="image" src="https://github.com/user-attachments/assets/f349c634-b29e-4af9-ac7a-3c64ffbe7b84">